lib/node_modules/@stdlib/blas/base/caxpy/README.md

```diff
@@ -205,51 +205,51 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
 #include "stdlib/blas/base/caxpy.h"
 ```
 
-#### c_caxpy( N, ca, \*CX, strideX, \*CY, strideY )
+#### c_caxpy( N, alpha, \*CX, strideX, \*CY, strideY )
```

Comment:
@ShabiShett07 You should also change `CX` to `X` and `CY` to Y`. Applies here and elsewhere.